server/org.wso2.developerstudio.eclipse.carbonserver44/src/org/wso2/developerstudio/eclipse/carbonserver44/internal/CarbonServerBehavior44.java

```diff
@@ -248,7 +248,10 @@ protected void addServletTransportPorts(List<Integer> ports, String carbonXmlPat
 			int offSet = Integer.parseInt((String)xPath.evaluate("/Server/Ports/Offset", document, XPathConstants.STRING));
 //			XPathExpression  xPathExpression=xPath.compile("/:Server/:Ports/:ServletTransports/:HTTPS");
 //			String evaluate = xPathExpression.evaluate(inputSource);
-			String evaluate = (String)catelineXPath.evaluate("/Server/Service/Connector[@sslProtocol=\"TLS\"]/@port", catelinaDocument, XPathConstants.STRING);
+			String evaluate = (String) catelineXPath
+					.evaluate(
+							CarbonServerConstants.CATALINA_XPATH_EXPRESSION_FOR_SSL_ENABLED_PORT,
+							catelinaDocument, XPathConstants.STRING);
```

Comment:
When formatting don't rely on the IDE to do the Job. If the IDE does not format correctly, please correct them manually. The formatting has to be corrected here.